pkg/spanconfig/spanconfigtestutils/utils.go

```diff
@@ -247,7 +247,21 @@ func ParseStoreApplyArguments(t *testing.T, input string) (updates []spanconfig.
 // the form "[start,end)". The span is assumed to have been constructed by the
 // ParseSpan helper above.
 func PrintSpan(sp roachpb.Span) string {
-	return fmt.Sprintf("[%s,%s)", string(sp.Key), string(sp.EndKey))
+	s := []string{
+		sp.Key.String(),
+		sp.EndKey.String(),
+	}
+	for i := range s {
```

Comment:
This is breaking symmetry with ParseSpan, worth at least updating the comment above.  Typo below "qe".